packs/actions/solicitation.ts

```diff
@@ -0,0 +1,139 @@
+import type { AppContext } from "$store/apps/site.ts";
+import { createClient } from "https://esm.sh/v135/@supabase/supabase-js@2.7.0";
+import {
+  RESPONSE_RISK3_APPROVED,
+  SOLICITATION_ENTITY_NAME,
+  STATUS_ENUM_ABLE,
+  STATUS_ENUM_ACCOUNT_OPENING,
+  STATUS_ENUM_DISAPPROVED,
+  STATUS_ENUM_IN_OPERATION,
+  STATUS_ENUM_SIGNATURE,
+  STATUS_ENUM_SUSPENDED,
+} from "deco-sites/niivu-bank/packs/utils/constants.ts";
+
+interface Fields {
+  phone: string;
+  zip_code: string;
+  street: string;
+  number: string;
+  complement?: string;
+  city: string;
+  state: string;
+  email: string;
+  rg?: string;
+  full_name?: string;
+  cpf?: string;
+  business_name?: string;
+  cnpj?: string;
+}
```

Comment:
Se voce dividir os campos do CPF e do CNPJ sua tipagem fica mais resiliente, por exemplo:
```suggestion
interface CNPJ {
  type: 'cnpj'
  business_name?: string;
  cnpj?: string;
}
```